api/v1alpha3/tags.go

```diff
@@ -30,7 +30,7 @@ type Tags map[string]string
 
 // Equals returns true if the tags are equal.
 func (t Tags) Equals(other Tags) bool {
```

Comment:
We could probably remove the `Equals` function on `Tags` as well as it adds no value (i.e. its not doing any custom equality logic).

Reply:
Makes sense @richardcase, but should a new PR be opened for this or this can be done in this one only ?
Like this PRs area was to just replace instances of reflect.DeepEqual().

In new PR I can remove this function in tags.go, after checking its call hierarchy and stuff to remove all errors related to it.
What do you suggest ?

Reply:
Yes @Ankitasw you are right, I was just worried about managing the function's call hierarchy. But it seems it is not being called anywhere in codebase.

I will commit no issues.

Reply:
@VibhorChinda are you sure it's not used anywhere in the codebase?
@richardcase in that case shall we get rid of this function?

Reply:
yes @Ankitasw I saw call hierarchy for this function. No location in codebase appears where it is being called.

Reply:
> We could probably remove the Equals function on Tags as well as it adds no value (i.e. its not doing any custom equality logic).

@VibhorChinda I think @richardcase is suggesting to get rid of this function by above comment, please go ahead and remove it.

Reply:
If there are no references then we can remove it then :)